### PR TITLE
fix: dont base64 encode rejection reasons

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: src/lib/plugin.js [hunk 1]
```diff
@@ -536,7 +536,9 @@ class FiveBellsLedger extends EventEmitter2 {
             relatedResources.cancellation_condition_fulfillment)
         } else if (fiveBellsTransfer.state === 'rejected') {
           const rejectedCredit = find(fiveBellsTransfer.credits, 'rejected')
-          const rejectionMessage = rejectedCredit ? rejectedCredit.rejection_message : 'transfer timed out.'
+          const rejectionMessage = rejectedCredit
+            ? new Buffer(rejectedCredit.rejection_message, 'base64').toString()
```
Comment (emschwartz) [inline src/lib/plugin.js:540]: You should use Buffer.from (see https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe)

Comment (sentientwaffle) [reply src/lib/plugin.js:540]: That causes the FTs to fail when run against older versions of node.

Comment (emschwartz) [reply src/lib/plugin.js:540]: Hmm. I think not using it though would expose the DoS vulnerability that caused that switch in the first place. What if I set the `rejection_message` to `10000000000000`?

Comment (sentientwaffle) [reply src/lib/plugin.js:540]: Ok, so we need to run the circleci tests with node 6, and then use `Buffer.from`.

Comment (sentientwaffle) [reply src/lib/plugin.js:540]: New issue: https://github.com/interledger/five-bells-integration-test/issues/39

**Comment by emschwartz** [comment]
LGTM

**Pull request merged**